app/controllers/admin/products_controller.rb

```diff
@@ -17,15 +17,19 @@ def show
     # GET /products/new
     def new
       @product = Product.new
```

Comment:
shouldn't need this line. it is created by cancancan with the load_resource statement above